pom.xml

```diff
@@ -823,6 +823,17 @@
                 <artifactId>maven-enforcer-plugin</artifactId>
                 <version>3.1.0</version>
                 <executions>
+                    <execution>
+                        <id>no-duplicate-declared-dependencies</id>
```

Comment:
Makes me wonder if we shouldn't make plans to ban dependency convergence to encourage more discipline on dependency management.